README.md

```diff
@@ -1 +1,164 @@
-# space-travelers-hub
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Space Traveler's Hub</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Traveler's Hub <a name="about-project"></a>
+
+<b>Space Traveler's Hub</b> is a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- #### [Live link](https://space-travelers-hub-zxf9.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+
+```
+$ node --version && npm --version
+```
+
+### Setup
+
+Clone this repository using the GitHub link provided below.
```

Comment:
- Please provide the github link users are to use in order to clone your project, this would help make your repo look even more professional 👍

Reply:
@mckent05 Added the link.